contents/blog/measuring-feature-success.md

```diff
@@ -0,0 +1,114 @@
+---
+title: How we measure feature success at PostHog
+date: 2023-04-20
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/open-source-testing-tools/testinghog.png
+featuredImageType: full
+category: Inside PostHog
+tags:
+ - Guides
+ - Feature flags
+---
+
+When it comes to new features, success can be tough to define. Although a business can succeed as a whole, what it means for a feature to succeed is less clear.
+
+At PostHog, we care about enabling teams to understand and measure feature success (we have [an entire team dedicated to it](/handbook/small-teams/feature-success)). As part of helping you do this, this post details what feature success is, and how we measure it at PostHog. We go through the real launch of a new feature (sampling) and explain how we measured success along the way.
+
+## What is feature success?
+
+Measuring feature success starts after deciding to work on a feature. We decide to build a feature based on quantitative and qualitative feedback combined with our thoughts on what’s best. Once we decide to build a feature, we start to judge its success based on if it:
+
+1. Solves a user problem. Key to this is understanding and defining users’ problems, then figuring out ways to solve them that fit with the product and business. 
+2. Is actually used. No matter how good you think your feature is, if it doesn’t get used, it isn’t benefitting users.
+
+Feature success is the process of enabling, measuring, and evaluating both of these. It is planning for how to develop a successful feature, a system for measuring its ongoing usage, and a process for evaluating its success.
+
+## How we measure feature success at PostHog
+
+Measuring feature success at PostHog is a multi-step process that goes hand-in-hand with our release process. The multi-step approach to feature success ensures we are building the right solution and that users actually care and use the solution we are building.
+
+To help you connect this to reality, we will follow along with how [Yakko](/handbook/company/team) recently built, measured, and evaluated [sampling](/docs/product-analytics/sampling).
+
+> **What is sampling?** The sampling feature speeds up queries by analyzing a portion of the data and extrapolating the results. Yakko decided to work on this feature because slow queries were causing frustrations, sampling felt like an intuitive solution, and manually benchmarking the impact of sampling on large customers confirmed this intuition (3-10x improvements).
+
+![Sampling in action](../images/blog/measuring-feature-success/sampling.gif)
+
+### 1. Build it and make sure it isn’t broken
+
+Once we decide to build a feature, we move fast and build it, review that it matches what we envisioned, and ship it behind a feature flag. The flag rolls out to a single developer to start. This helps them to check if it has the smallest bit of success in solving the problem in production.
```

Comment:
Before you start building something, shouldn't you first define why theres a need for it, prioritize it and then define a criteria for evaluating it's success?

Reply:
The draft this was inspired by does cover this so should be easy to pop it in here if you like @ivanagas 

https://github.com/PostHog/posthog.com/blob/9675ce3fa70941dfbde7f8cf2ba63110d1b49bb9/contents/blog/building-a-new-feature-e2e.md#should-we-build-this
https://github.com/PostHog/posthog.com/blob/9675ce3fa70941dfbde7f8cf2ba63110d1b49bb9/contents/blog/building-a-new-feature-e2e.md#v0

Reply:
Added, thanks!